resolwe_bio/processes/expression/dicty_bcm.yml

```diff
@@ -80,67 +80,37 @@
     program: |
       NAME=`basename {{ alignment.bam.file }} .bam`
 
-      function compute_coverage {
```

Comment:
A whole lot of stuff has been removed... How is this connected to the change of Docker image?

Reply:
Instead of porting obsolete functionality of the [xtabcoverage](https://github.com/genialis/resolwe-bio/pull/697/files#diff-55ccdfababdae6fb0b80c6dc8171d791L1) script to Python3, I decided to remove this part when switching Docker images. Additionally, this part of the code was the source of errors with some "non-standard" genome/annotation inputs.